common/src/main/java/com/box/l10n/mojito/slack/SlackClients.java

```diff
@@ -0,0 +1,40 @@
+package com.box.l10n.mojito.slack;
+
+import java.util.AbstractMap.SimpleEntry;
+import java.util.Map;
+import java.util.stream.Collectors;
+import org.springframework.stereotype.Component;
+
+/**
+ * Create Slack clients from configuration and expose them by client ids. This allows to configure
+ * and communicate with different Slack instances.
```

Comment:
Could you remind me please for which scenario we now require the ability to communicate with different Slack instances (vs. just assuming one instance) ?